packages/components/image/src/image.vue

```diff
@@ -67,17 +71,18 @@ const emit = defineEmits(imageEmits)
 
 let prevOverflow = ''
 
+const { loading } = toRefs(props)
 const { t } = useLocale()
 const ns = useNamespace('image')
 const rawAttrs = useRawAttrs()
 const attrs = useAttrs()
 
+const imageSrc = ref('')
 const hasLoadError = ref(false)
-const loading = ref(true)
-const imgWidth = ref(0)
-const imgHeight = ref(0)
+const isLoading = ref(true)
 const showViewer = ref(false)
 const container = ref<HTMLElement>()
+const supportLoading = ref(true)
```

Comment:
it's a constant, and never change. 
```suggestion
const supportLoading = 'loading' in HTMLImageElement.prototype
```

Reply:
Because must make sure to judge in the browser environment, otherwise maybe get an error in server rendering.

Reply:
Hmmm you can use `isServer` / `isClient` to handle this.